.gitignore

```diff
@@ -14,6 +14,12 @@ node_modules
 
 # Yarn Integrity file
 .yarn-integrity
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
```

Comment:
Will this list ever update? Is there some documentation around what should and should not be ignored?

Reply:
those are the defaults that `yarn init` creates. so i added with them because seems to want you to. i'll add a comment to that effect. they're def not all used as things stand, so maybe they should be pruned?